tests/phpunit/CiviTest/CiviUnitTestCase.php

```diff
@@ -2547,12 +2548,10 @@ protected function eventPriceSetCreate($feeTotal, $minAmt = 0, string $type = 'T
     }
     $priceFieldID = $this->callAPISuccess('PriceField', 'create', $paramsField)['id'];
     $this->ids['PriceField']['event_' . strtolower($type)] = $priceFieldID;
-    $this->_ids['price_field'] = [$priceFieldID];
```

Comment:
This might cause test fails - but it is extraneous so I will fix the tests if so. PHP 8 wouldn't like this